heartbeat/monitors/wrappers/summarizer/plugmondur.go

```diff
@@ -65,13 +65,14 @@ func (bwdsp *BrowserDurationPlugin) EachEvent(event *beat.Event, _ error) EachEv
 }
 
 func (bwdsp *BrowserDurationPlugin) BeforeSummary(event *beat.Event) BeforeSummaryActions {
-	// do not set the duration if we're missing data
-	// usually because the runner failed in a weird way
+	var durUS int64
 	if bwdsp.startedAt == nil || bwdsp.endedAt == nil {
-		return 0
+		durUS = 0
```

Comment:
Given we figured out the Kibana bug is not related to this one. I think we have to do two things
1. If startAt was nil, Never ran any journey - duration can be set to 0
2. if endAt is nil, Synthetics event exited without sending `journey/end`, we need to handle it and rely on event.Timestamp for the endAt value. Should not be 0.

Reply:
I've pushed this change